src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -150,9 +152,15 @@
     "manufacturersXml": "../../../../src/app/zap-templates/zcl/data-model/manufacturers.xml",
     "options": {
         "text": {
-            "defaultResponsePolicy": ["Always", "Conditional", "Never"]
+            "defaultResponsePolicy": [
```

Comment:
This change is making your PR fail CI, quite apart from making it harder to review.

Reply:
I can probably fix this; just alchemy's JSON formatting out of sync with what restyled expects.